src/qml/editorwidgets/Range.qml

```diff
@@ -146,20 +147,26 @@ EditorWidgetBase {
   }
 
   function decreaseValue() {
-      if ( textField.text ) {
-          var newValue = textField.text * 1 - rangeItem.step;
-          textField.text = Math.max( rangeItem.from, newValue );
+      var currentValue = textField.text
+      var newValue
+      if ( currentValue ) {
+          newValue = currentValue * 1 - rangeItem.step;
```

Comment:
```suggestion
      var currentValue = isFinite(+textField.text)
      var newValue
      if ( !isNaN(currentValue) ) {
          newValue = currentValue * 1 - rangeItem.step;
```
I think this is safer. 

Note I have changed from `textField.text * 1` to `+textField.text`, which some claim is slower, but for sure much more common approach to change numeric string to number.

Reply:
but tbh since not all QField developers are aware of JS quircks, `var currentValue = isFinite(Number(textField.text))` is the best.

Reply:
@suricactus , thanks, I've taken your comments into account. I'm not initializing var currentValue with a Number.isFinit(...) because I actually want that numerical value, not a boolean :) the isNaN function is a good trick, TIL.